source/jsonnet/northern-ireland/individual/blocks/identity-and-health/no_religion.jsonnet

```diff
@@ -83,6 +83,12 @@ local question(title) = {
         when: [rules.under3],
       },
     },
+    {
+      goto: {
+        block: 'health',
+        when: [rules.lastBirthdayAgeUnder(3)],
+      },
+    },
```

Comment:
Both these rules need when rules to check the no-religion-answer was selected

Reply:
this is wrong in the quick launch - being under 3 then selecting none is taking me to health instead of language

Reply:
This seems to be correct. To get to this block, you must have either, said `None` to region answer or, not answered it.